src/Command/LogCommand.php

```diff
@@ -46,7 +46,7 @@ public function __construct (CronJobRegistry $registry, CronModel $model)
     protected function configure () : void
     {
         $this
-            ->addOption("single", null, InputOption::VALUE_NONE, "Whether the log for a single job should be shown.");
+            ->addOption("single", null, InputOption::VALUE_OPTIONAL, "Whether the log for a single job should be shown.", false);
```

Comment:
We should re-word the help text to indicate that a filter-like functionality is available.

```suggestion
            ->addOption("single", null, InputOption::VALUE_OPTIONAL, "Whether the log for a single job should be shown. You can optionally provide a cronjob key to only show its logs.", false);
```